modules/ROOT/pages/expressions/predicates/label-expression-predicates.adoc

```diff
@@ -0,0 +1,249 @@
+
+image::label_expression_predicates_graph.svg[width="700",role="middle"]
+
+To recreate the graph, run the following query against an empty Neo4j database:
+
+[source, cypher]
+----
+CREATE (alice:Person&Manager {name:'Alice', age: 65, skills: ['Java', 'Python']}),
+       (cecil:Person&Developer {name: 'Cecil', age: 25, skills: ['Java', 'Python']}),
+       (cecilia:Person&Developer {name: 'Cecilia', age: 31, skills: ['JavaScript', 'TypeScript']}),
+       (charlie:Person&Engineer {name: 'Charlie', age: 61, skills: ['C++', 'Python']}),
+       (daniel:Person&Director {name: 'Daniel', age: 39, skills: ['JavaScript', 'Slides']}),
+       (eskil:Person&CEO {name: 'Eskil', age: 39, skills: ['Slides', 'ChatGPT']}),
+
+       (cecil)-[:WORKS_FOR]->(alice),
+       (cecilia)-[:WORKS_FOR]->(alice),
+       (charlie)-[:WORKS_FOR]->(daniel),
+       (alice)-[:REPORTS_TO]->(daniel),
+       (daniel)-[:REPORTS_TO]->(eskil)
+----
+
+[node-label-expression-predicate]
+== Test that node has a certain labels
+
+Given `p` is a node, `p:Manager` tests whether node `p` has the label `Manager` or not and results in `true` or `false`, respectively.
+
+// tag::node-label-expression-predicate-1[]
+[source, cypher]
+----
+MATCH (p:Person)
+RETURN p.name AS name, p:Manager AS isManager
+----
+// end::node-label-expression-predicate-1[]
+
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| name | isManager
+
+| "Alice" | true
+| "Cecil" | false
+| "Cecilia" | false
+| "Charlie" | false
+| "Daniel" | false
+| "Eskil" | false
+
+2+d|Rows: 6
+|===
+
+Given `p` is a node and a more complex xref::patterns/reference.adoc#label-expressions[label expression], e.g, `Manager|Director|CEO`, `p:Manager|Director|CEO` tests whether node `p` matches the label expression `Manager|Director|CEO` or not and results in `true` or `false`, respectively.
+More specifically, `p:Manager|Director|CEO` tests whether `p` has at least one of the three labels `Manager`, `Director`, and `CEO`.
+
+// tag::node-label-expression-predicate-2[]
+[source, cypher]
+----
+MATCH (p:Person)
+RETURN p.name AS name,
+       p:Manager|Director|CEO AS isManager
+----
+// end::node-label-expression-predicate-2[]
+
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| name | isManager
+
+| "Alice" | true
+| "Cecil" | false
+| "Cecilia" | false
+| "Charlie" | false
+| "Daniel" | true
+| "Eskil" | true
+
+2+d|Rows: 6
+|===
+
+`p:!CEO` tests whether `p` does not have the label `CEO`.
+
+[source, cypher]
+----
+MATCH (p:Person)-[r]->(m)
+RETURN p.name AS name,
+       m:!CEO AS doesNotWorkDirectlyForTheCEO
+----
+
+[role="queryresult",options="header,footer",cols="2*<m"]
+|===
+| name | doesNotWorkDirectlyForTheCEO
+
+| "Alice" | true
+| "Cecil" | true
+| "Cecilia" | true
+| "Charlie" | true
+| "Daniel" | false
+
+2+d|Rows: 5
+|===
+
+When '<exp>', then the label expression predicate results in `null`, e.g. if `p` is `null`, then `p:!CEO` results in `null`.
```

Comment:
It feels like something is missing here, it should not say just When '\<exp\>' right?